node-js-basics-101/node_modules/parseuri/test.js

```diff
@@ -0,0 +1,51 @@
+var assert = require('better-assert');
+var expect = require('expect.js');
+var parseuri = require('./index.js');
+
+describe('my suite', function(){
+	it('should parse an uri', function () {
+    var http = parseuri('http://google.com')
+      , https = parseuri('https://www.google.com:80')
+      , query = parseuri('google.com:8080/foo/bar?foo=bar')
+      , localhost = parseuri('localhost:8080')
+      , ipv6 = parseuri('2001:0db8:85a3:0042:1000:8a2e:0370:7334')
+      , ipv6short = parseuri('2001:db8:85a3:42:1000:8a2e:370:7334')
+      , ipv6port = parseuri('2001:db8:85a3:42:1000:8a2e:370:7334:80')
+      , ipv6abbrev = parseuri('2001::7334:a:80')
+      , ipv6http = parseuri('http://[2001::7334:a]:80')
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->